.travis.yml

```diff
@@ -6,15 +6,15 @@ services:
   - mysql
   - postgresql
 rvm:
-  - 1.8.7
-  - 1.9.3
-  - 2.0.0
-  - 2.1
+  - 2.2
+  - 2.3
+  - 2.4
+  - 2.5
```

Comment:
You may also want to update the README accordingly :wink:

Reply:
@ZeroPointEnergy would you mind making a PR for those fixes too?  I have a feeling we can get them merged now.